fe/fe-core/src/main/java/com/starrocks/catalog/ResourceGroup.java

```diff
@@ -40,6 +39,7 @@ public class ResourceGroup implements Writable {
     public static final String SOURCE_IP = "source_ip";
     public static final String DATABASES = "db";
     public static final String CPU_CORE_LIMIT = "cpu_core_limit";
```

Comment:
What's the difference between `CPU_CORE_LIMIT` and `MAX_CPU_CORES`? One is soft limit, one is hard limit?

Reply:
- `CPU_CORE_LIMIT` is only the weight of scheduling. This name is misunderstanding, and we will rename it to `cpu_weight` in the future.
- `MAX_CPU_CORES` is hard limit. 
    - FE uses it to queue queries of a group, whose cpu usage runtime exceeds `MAX_CPU_CORES` in a period.
    - BE will use it to avoid scheduling tasks of a group in the future, whose cpu usage runtime exceeds `MAX_CPU_CORES` in a period.

Reply:
I would write document in the next PR.